src/components/panel/panel.spec.js

```diff
@@ -105,6 +108,62 @@ describe('$mdPanel', function() {
     expect(PANEL_EL).not.toExist();
   });
 
+  it('should call onDomAdded if provided when adding the panel to the DOM', function() {
```

Comment:
These tests should be in the `config options:` describe block, not before the basic functionality tests for showing and hiding a panel.